Makefile

```diff
@@ -108,5 +108,9 @@ docker_image_gradle: compileWithGradle
 	cp dist/* deployment/lib
 	./build/build_docker_image.sh
 
-compileWithGradle:
+compile-with-gradle:
 	./gradlew buildAll
+
+build-docker-image: compile-with-gradle
+	./gradlew buildAll
```

Comment:
With the multistage build you don't need to do the grade build first.

Reply:
Thanks, deleted that